lib/doorkeeper/errors.rb

```diff
@@ -38,6 +38,18 @@ def type
       end
     end
 
+    class MissingClientId < DoorkeeperError
```

Comment:
What if introduce one class like `MissedRequiredParameter` with overridden initializer that receives missed parameter? I think it will be easier to introduce error description and will be more DRY, WDYT ?

```ruby
class MissedRequiredParameter < DoorkeeperError
  def initialize(message, parameter)
    super(message)
    @parameter = parameter
  end

  def type
     :invalid_request
  end
 
  # ...

Reply:
Great suggestion, I will try this to inform missing parameter.